packages/cli/src/databases/entities/ExecutionEntity.ts

```diff
@@ -49,7 +50,7 @@ export class ExecutionEntity {
 	@Column({ type: datetimeColumnType, nullable: true })
 	stoppedAt: Date;
 
-	@Column(datetimeColumnType)
+	@DeleteDateColumn({ type: datetimeColumnType, nullable: true })
 	deletedAt: Date;
```

Comment:
should this be 
```suggestion
	deletedAt?: Date;
```

Reply:
Not 100% on this, but I logged on startup webhook entities (which have `webhookId` and `pathLength` which are nullable are marked with `?`) and they show up as `null` when missing:

```
> n8n@1.7.0 start /Users/ivov/Development/n8n
> run-script-os


> n8n@1.7.0 start:default
> cd packages/cli/bin && ./n8n

n8n ready on 0.0.0.0, port 5678
Initializing n8n process
Version: 1.7.0
 ================================
   Start Active Workflows:
 ================================
   - My workflow 21 (ID: HDemWLl4i60bkGsm)
allWebhooks [
  WebhookEntity {
    workflowId: 'HDemWLl4i60bkGsm',
    webhookPath: '9347adb3-289c-4508-8705-531575ac3e60',
    method: 'GET',
    node: 'Webhook',
    webhookId: null,
    pathLength: null
  }
]
```

So if anything no properties that are nullable columns should have `?` right?